tck/pom.xml

```diff
@@ -112,6 +112,12 @@
             <version>${version.glassfish.json}</version>
         </dependency>
 
+        <dependency>
+            <groupId>com.github.java-json-tools</groupId>
+            <artifactId>json-schema-validator</artifactId>
+            <version>2.2.10</version>
+        </dependency>
```

Comment:
Did you try to exclude some of the dependencies here to limit extra lib and CQ ?

Reply:
no, but I can try, is there something I can help you with the filing of the CQ?